src/components/Popover/Popover.tsx

```diff
@@ -57,6 +57,7 @@ export const Popover = React.forwardRef<PopoverInstanceProps, PopoverProps & QAP
         focusTrap,
         autoFocus,
         restoreFocusRef,
+        enableButtonEmulation = true,
```

Comment:
Shouldn't be it always enabled?

Reply:
No, as Popover can wrap `<button>` element, which emulates click event on Enter pressing. After that the browser emulates click event, so no need to emulate click event twice

Reply:
reverted changes. There is no longer `enableButtonEmulation` prop